SourceKitStressTester/Sources/Common/Message.swift

```diff
@@ -52,25 +47,26 @@ public enum SourceKitError: Error {
       return request
     case .timedOut(let request):
       return request
-    case .softTimeout(let request, _, _):
-      return request
     case .failed(_, let request, _):
       return request
     }
   }
 
   /// Soft errors are allowed to match XFails, but don't fail the stress tester
   /// on their own.
-  /// The current use case for soft errors are soft timeouts, where the request
-  /// took more than half of the allowed time. If the issue is XFailed, we don't
-  /// want to mark it as unexpectedly passed because the faster execution time
-  /// might be due to noise. But we haven't surpassed the limit either, so it
-  /// shouldn't be a hard error either.
+  ///
+  /// The current use case for soft errors are timeouts, since it's possible
+  /// that it hits a failure on a faster run, which can lead to non-determinism
+  /// in matching XFAILs.
```

Comment:
This comment is out-of-date now, right?

Reply:
We still need soft errors for *proper* timeouts, since on faster runs a request can hit an error, and on slower runs they can hit a timeout. The [current stress tester failure](https://ci.swift.org/job/swift-main-sourcekitd-stress-tester/1051/) is because we have requests timing out, but sometimes they finish quickly and hit a known issue. So we need to have timeouts be able to match XFAILs.